testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -73,9 +73,12 @@ class BatchReport extends React.Component {
       report: processedReport,
       filteredReport,
       selectedUIDs,
-      lastManualSelectedUIDs: selectedUIDs,
       loading: false,
       logs: report.logs,
+    }, () => {
+      this.props.history.replace(
```

Comment:
What is this history used for?

Reply:
history representiung the browser history, it is injected by the router. If you push or replace things in it that made the browser to actuially navigate. Push equivalent of redirect while replace is similar, just the original url is removed from history and replaced with the new one.